docs/_templates/layout.html

```diff
@@ -89,3 +89,25 @@
+            <a href="https://nl.wagtail.space/">
+                <div>
+                    <strong>Wagtail Space NL</strong>
+                    <br>12 &ndash; 14 June 2024 in Arnhem, The Netherlands
+                </div>
+            </a>
+            <a href="https://us.wagtail.space/">
+                <div>
+                    <strong>Wagtail Space US</strong>
+                    <br>June 2024 in Philadelphia, USA
+                </div>
+            </a>
```

Comment:
I think most people see banners as one big clickable area. So, is it obvious enough that these are two sparate links? Do we need some visual distinction? Maybe a clear hover state is enough? 

And, does the US team want to advertise the event? Even if the date isn't final yet? Seems a bit strange to me to grab peoples attention with something that is not yet actionable.

Reply:
They do have a hover state that's separate.

Also the Wagtail Space US link is live with a form to sign up for info about the 2024 event.

https://us.wagtail.space/

Reply:
I also think that they are quite isolated visually and have clear headers.

Alternatively we could update back to one link with both bits of information and a link just to the main Wagtail space website.

Reply:
The original was better. If the links have hover states, that should suffice.

Cool! Thank you for the efforts!